wiki/en/en-Contribution.md

```diff
@@ -25,7 +25,7 @@ Have a look at the [CONTRIBUTING file](https://github.com/jamulussoftware/jamulu
 
 **The following content is for the documentation on jamulus.io only, not for the Jamulus software!**
 
-If you aren't comfortable with Pull Requests, commits, etc. just open an issue. We will then care about your changes. If you want to do the changes on your own and open a Pull Request, follow those instructions:
+If you aren't comfortable with Pull Requests, commits, etc. just open an issue. We will then care about your changes. If you want to do the changes on your own and open a Pull Request, follow these instructions:
```

Comment:
'We will then take care of your changes.'

Reply:
Done. We have to decide what to do with this part. Do we really need to be that specific?